packages/redux-simple-auth/package.json

```diff
@@ -53,10 +53,9 @@
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-jsx-a11y": "^5.0.3",
     "eslint-plugin-react": "^7.0.1",
-    "husky": "^0.13.3",
     "jest": "^22.1.4",
     "jest-fetch-mock": "^1.4.1",
-    "lint-staged": "^3.4.2",
+    "lint-staged": "^6.1.1",
```

Comment:
I've fixed this issue in this PR https://github.com/jerelmiller/redux-simple-auth/pull/41

Reply:
I'll pull that commit out then :)